README.md

```diff
@@ -12,7 +12,7 @@ php-WebHDFS is a PHP client for [WebHDFS](http://hadoop.apache.org/docs/r2.0.3-a
 ## Install via composer
 
 ```bash
-composer require simpleenergy/php-webhdfs
+composer require dreamfactory/php-webhdfs
```

Comment:
Changes look good, but I think we should still keep the original package name. I'll be glad to merge your changes as a new version if you remove this.